docs/en/02_Developer_Guides/14_Files/01_Image.md

```diff
@@ -18,60 +18,81 @@ Allows upload of images through limiting file extensions with `setAllowedExtensi
 
 See [Topics: Rich Text Editing](/topics/rich-text-editing).
 
-### Resizing Images in PHP
+### Manipulating images in Templates
 
-The following are methods defined on the GD class which you can call on Image Objects. Note to get the following to work
-you need to have GD2 support in your PHP installation and because these generate files you must have write access to
-your tmp folder. 
+You can manipulate images directly from templates to create images that are
+resized and cropped to suit your needs.  This doesn't affect the original
+image or clutter the CMS with any additional files, and any images you create
+in this way are cached for later use. In most cases the pixel aspect ratios of
+images are preserved (meaning images are not stretched).
+
+Here are some examples, assuming the `$Image` object has dimensions of 200x100px:
 
 	:::php
-	// manipulation functions
-	$image->resize(width,height); // Basic resize, just skews the image
-	$image->resizeRatio(width,height) // Resizes an image with max width and height
-	$image->paddedResize(width,height) // Adds padding after resizing to width or height.
-	$image->croppedImage(width,height) // Crops the image from the centre, to given values.
-	$image->resizeByHeight(height) // Maximum height the image resizes to, keeps proportion
-	$image->resizeByWidth(width) // Maximum width the image resizes to, keeps proportion 
-	$image->greyscale(r,g,b) // alters image channels ===
+	// Scaling functions
+	$Image.ScaleWidth(150) // Returns a 150x75px image
+	$Image.ScaleMaxWidth(100) // Returns a 100x50px image (like ScaleWidth but prevents up-sampling)
+	$Image.ScaleHeight(150) // Returns a 300x150px image (up-sampled. Try to avoid doing this)
+	$Image.ScaleMaxHeight(150) // Returns a 200x100px image (like ScaleHeight but prevents up-sampling)
+	$Image.Fit(300,300) // Returns an image that fits within a 300x300px boundary, resulting in a 300x150px image (up-sampled)
+	$Image.FitMax(300,300) // Returns a 200x100px image (like Fit but prevents up-sampling)
+	
+	// Cropping functions
+	$Image.Fill(150,150) // Returns a 150x150px image resized and cropped to fill specified dimensions (up-sampled)
+	$Image.FillMax(150,150) // Returns a 100x100px image (like Fill but prevents up-sampling)
```

Comment:
I definitely think this is a keeper, because what if you request a Fill(2000,1000) on an image that is only 500x300px? You get a 2000x1000px image that has been greatly upsampled, where as FillMax will give you a 500x350px image.